src/libs/ReportUtils.ts

```diff
@@ -1396,26 +1225,23 @@ function getMoneyRequestSpendBreakdown(report, allReportsDict = null) {
 
 /**
  * Get the title for a policy expense chat which depends on the role of the policy member seeing this report
- *
- * @param {Object} report
- * @param {Object} [policy]
- * @returns {String}
  */
-function getPolicyExpenseChatName(report, policy = undefined) {
-    const reportOwnerDisplayName = getDisplayNameForParticipant(report.ownerAccountID) || lodashGet(allPersonalDetails, [report.ownerAccountID, 'login']) || report.reportName;
+function getPolicyExpenseChatName(report: OnyxEntry<Report>, policy: OnyxEntry<Policy> | undefined = undefined): string | undefined {
```

Comment:
```suggestion
function getPolicyExpenseChatName(report: OnyxEntry<Report>, policy?: OnyxEntry<Policy>): string | undefined {
```

Reply:
So actually I did it on purpose I think we always want users of this function to pass a `policy` param
but it can be undefined based on this article https://tkdodo.eu/blog/optional-vs-undefined

Reply:
I got you here and agree. I still think that default value `undefined` is extra here. Because JS applied default values when income data type is `undefined`, so looks strange.

```suggestion
function getPolicyExpenseChatName(report: OnyxEntry<Report>, policy: OnyxEntry<Policy> | undefined): string | undefined {
```

Reply:
yeah true , but in several places those parameters will have after some other parameters with default values and when I will remove default values the linter will complain 😞